cli/js/lib.deno.ns.d.ts

```diff
@@ -396,6 +396,16 @@ declare namespace Deno {
     }
   ): IterableIterator<Uint8Array>;
 
+  /** Turns a ReadableStream into a Reader
+   * @param src The RedableStream to convert
+   */
+  export function toReader(src: ReadableStream): Reader;
+
+  /** Turns a WritableStream into a Writer
+   * @param dst The WritableStream to convert
+   */
+  export function toWriter(dst: WritableStream): Writer & Closer;
```

Comment:
New API additions should be unstable. Can you move these to lib.deno.unstable.d.ts ?

I'm not sure about the names in particular...

Reply:
Will do!. Regarding the names, I'm open to suggestions.

@nayeemrmn suggested:
- `fromStreamReader`
- `fromStreamWriter`